packages/mcp-docs-server/src/tools/examples.ts

```diff
@@ -52,14 +53,22 @@ export const examplesInputSchema = z.object({
     .describe(
       'Name of the specific example to fetch. If not provided, lists all available examples.' + examplesListing,
     ),
+  queryKeywords: z
```

Comment:
this is super interesting! did you notice agents using this more/less for certain cases?

Reply:
nice, well I'm stoked to try it out